requirements-lock.txt

```diff
@@ -7,9 +7,9 @@ httpcore==0.13.7
 httpx==0.18.1
 idna==3.3
 pydantic==1.9.0
-reasoner-pydantic==2.1.0
+reasoner-pydantic==2.2.1
```

Comment:
A new version of reasoner-pydantic will be out very soon. We might want to wait to merge this until after upgrading.